backend/actions/address_detail.js

```diff
@@ -0,0 +1,45 @@
+const Client = require('bitcoin-core');
+const app = require('../app.js');
+
+const cl = new Client({
+  network: 'regtest',
+  username: 'user',
+  password: 'password',
+  port: 18443,
+});
+
```

Comment:
Do not write credentials directly. If you want to run it locally, you can also specify the path of the tapyrus configuration file.

Reply:
ok i will make the changes

Reply:
In this implementation, the path is fixed, so it is environment-dependent. Therefore, it is better to read the path from the environment variable.

Reply:
ok i changed to use the cross-env package